src/client.cpp

```diff
@@ -64,21 +64,31 @@ void waybar::Client::handleOutput(struct waybar_output &output) {
 }
 
 bool waybar::Client::isValidOutput(const Json::Value &config, struct waybar_output &output) {
-  bool found = true;
+
+  auto check_output = [](std::string_view config_output_name, struct waybar_output &output) {
+    if (!config_output_name.empty()) {
+      if (config_output_name.substr(0, 1) == "!") {
+          return config_output_name.substr(1) != output.name;
+      }
+      return config_output_name == output.name;
+    }
+    return false;
+  };
+
   if (config["output"].isArray()) {
-    bool in_array = false;
     for (auto const &output_conf : config["output"]) {
-      if (output_conf.isString() && output_conf.asString() == output.name) {
-        in_array = true;
-        break;
+      if (output_conf.isString()) {
```

Comment:
```suggestion
      if (output_conf.isString() && check_output(output_conf.asString(), output)) {
```

Reply:
Mb we should not do this check here, but only when only one screen name is passed, otherwise when there is
["!edp1", "edp2"] all screens will enter the config